.github/workflows/verify-schema.yml

```diff
@@ -1,19 +0,0 @@
-name: verify schema
```

Comment:
Why is this workflow being deleted?

Reply:
Haven't we moved the schema back to defsec? https://github.com/aquasecurity/defsec/pull/1517

Correct me if I'm wrong, as this PR removes the schema workflow and the schema itself from trivy-iac.